fame.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: page
+title: fame
+permalink: /fame/
+---
+Hall of Fame
+---
+
+<style>
+.project {
+  margin: 1em 0 0 0;
+  padding: 0 0 0 1em;
+}
+
+.title {
+    font-weight: bold;
+    font-family: 'Changa', sans-serif;
+    font-size: 24px;
+
+}
+
+.name {
+    padding-left: 25px;
+}
+
+hr {
+	height: 10px;
+	border: 0;
+	box-shadow: 0 10px 10px -10px #8c8b8b inset;
+    padding-bottom: 20px;
+}
+
+iframe {
+    padding-top: 15px;
+}
+
+</style>
+
+<div id="fame" />
+
+<script>
+
+var fame_data = {{ site.data.fame | jsonify }};
+
+var fame_div = d3.select('#fame');
+
+fame_div.selectAll('.project')
+  .data(fame_data)
+  .enter().append('div')
+  .attr('class', 'project')
+  .html( render_project )
+
+function render_project(d, i, A) {
+    var s = '';
+    s += '<link href="https://fonts.googleapis.com/css?family=Changa" rel="stylesheet">';
```

Comment:
can use es2015 template strings in modern browsers to clean this code up considerably: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals http://caniuse.com/#feat=template-literals

Reply:
oh, also, if you're including stylesheets you should do that in the document itself, not dynamically, or else users will get a bad FOUC